### ISSUE TITLE
Add a grid view to the projects page

### ISSUE THREAD

**Issue by krnekhelesh** [sonrisesoftware/project-dashboard#114]
The first page that a user sees when opening Project Dashboard is the projects page. The projects page at the moment is a bit dull. 
1. It would be nice if a grid view can be added where the projects could take the project's icon from github. This will also provide easier navigation since an icon generally helps identify items faster than reading the title.
2. If the user uses the current list view, then perhaps we should use a styling similar to the view used inside a project with plugins. Each ubuntushape could present as filters. For instance, I have three projects in my workflow which are Flashback, Project Dashboard and a Test Project. This can be filtered into projects owned by me and projects followed by me. Obviously this is one idea. I am sure we can come up with other filters if necessary.

**Comment by iBelieve** [comment]
Definitely something important to work on.

GitHub doesn't have an icon setting, so there's no way to pull in an icon from there. But the user will be able to add an icon manually, or it could be pulled in from Launchpad once that plugin gets done.

I have a couple ideas for how projects will be sorted and displayed, but I want to think through them a bit more.

**Comment by krnekhelesh** [comment]
Yeah I just wanted to get this out before I forget it :). Nice to know that you considered this as well.
